docs/sphinx/compiling/general.rst

```diff
@@ -209,11 +204,11 @@ Cmake build options
 Examples
 ---------
 
-To build with HSA, DPDK and debug symbols:
+To build with DPDK and debug symbols:
 
 .. code:: bash
 
-    cmake -DRTE_SDK=/opt/dpdk-stable-16.11.4/ -DRTE_TARGET=x86_64-native-linuxapp-gcc -DUSE_DPDK=ON -DUSE_HSA=ON -DCMAKE_BUILD_TYPE=Debug ..
+    cmake -DRTE_SDK=/opt/dpdk-stable-16.11.4/ -DRTE_TARGET=x86_64-native-linuxapp-gcc -DUSE_DPDK=ON -DCMAKE_BUILD_TYPE=Debug ..
```

Comment:
There isn't much point building with DPDK 16.11.4 on the newer Ubuntu systems.  We should just have the `-DUSE_DPDK=ON` flag and let it pick up the version from the Ubuntu apt install of DPDK.